tools/amd_build/patches/a_aten_src_THCUNN_BCECriterion.cu.patch

```diff
@@ -0,0 +1,27 @@
+diff --git a/aten/src/THCUNN/BCECriterion.cu b/aten/src/THCUNN/BCECriterion.cu
+index ccb40008c..1051a19af 100644
+--- a/aten/src/THCUNN/BCECriterion.cu
++++ b/aten/src/THCUNN/BCECriterion.cu
+@@ -9,6 +9,14 @@
+ #include <thrust/transform.h>
+ #include <thrust/transform_reduce.h>
+ 
++#if defined(__HIP_PLATFORM_HCC__)
++template <typename T>
++inline __host__ __device__ T eps();
++template <>
++inline __host__ __device__ float eps() { return 1e-12f; }
++template <>
++inline __host__ __device__ double eps() { return 1e-12; }
```

Comment:
This code looks like it's what's originally been there, except for the `__host__` addition. We could just patch the original file if that helps.